### ISSUE TITLE
Read response from BLE device

### ISSUE THREAD

**Issue by poonamalpha** [glairnarra31/bluetooth-le#1]
hi..how we can read response frame from ble device on write callback, can you help me out?

**Comment by orcunor** [comment]
If the property of your characteristic supports notify or indicate, you can read with the event like (GattCharacteristic_ValueChanged), if it does not support read, you can not read it.

**Comment by poonamalpha** [comment]
Yes got it, but how to read or extract response frame of BLE device which return by BLE device after writing some frame to BLE device using that characteristic(Notify & Indicate)?

**Comment by orcunor** [comment]
private void GattCharacteristic_ValueChanged(GattCharacteristic sender, GattValueChangedEventArgs args)
        {
            var reader = DataReader.FromBuffer(args.CharacteristicValue);
            byte[] input = new byte[reader.UnconsumedBufferLength];
            reader.ReadBytes(input);  
        }

You can trigger this event by connecting it to characeristic.ValueChanged if notify or indicate succes.

GattCharacteristic.ValueChanged += GattCharacteristic_ValueChanged;

you can do that like this. whenever there is an output you can read it with this event.
